doc/make.py

```diff
@@ -206,15 +206,15 @@ def _add_redirects(self):
 
                 try:
                     title = self._get_page_title(row[1])
-                except Exception:
+                except (docutils.utils.SystemMessage, FileNotFoundError):
```

Comment:
I understand that, but in this case I think it is better to keep the `Exception`. We don't have proper tests for this, only one sphinx version and one docutils version on a clean doc build on Azure on one OS, but people can build the docs in many different ways in practice. I don't think we any guarantee that it will not give other errors there. And in this case it is also not a crucial part (like causing bugs in results) if we would swallow exceptions incorrectly.

Reply:
@jbrockmendel maybe revert this

Reply:
reverted+green